canonical/specs-v1.canonical.com
# Migrate the client side to react.js

## Done

- Migrate the frontend code to react.js
- Fix miss parsed spec status: `pending approval` as shown as `unknown`
- Fix URL copy-paste restore: when sharing the URL, some of the filters weren't shared properly

## QA

- Make sure that you have the Google credentials
- Run the app:
```bash
dotrun
```
- Make sure that the app is still working
- Make sure that the different filtering options are working
- Make sure that the URL is updated with the filtering state
- Make sure that the URL with filtering is restored after copy pasting the URL to a new tab
